schedule/sles4sap/sles4sap_gnome_saptune.yaml

```diff
@@ -6,9 +6,26 @@ description: >
 vars:
   BOOTFROM: c
   BOOT_HDD_IMAGE: '1'
+  PUBLIC_CLOUD_RESOURCE_GROUP: 'HanaSR'
```

Comment:
How this one is used? How it is injected in the deployment process?  It is not part of sles4sap_azure_generic.yaml?

Reply:
> How this one is used? How it is injected in the deployment process? It is not part of sles4sap_azure_generic.yaml?

Good question, I remember if not set "PUBLIC_CLOUD_RESOURCE_GROUP" test case will be failed. But I can not find the failed openQA run currently.

Reply:
Maybe the new split qesapdeployment code don't need this variable.
I used 'tests/publiccloud/prepare_instance.pm" to create instance before. Let me try to delete it.

Reply:
Deleted. It is useless now: https://openqaworker15.qa.suse.cz/tests/67509#step/ssh_interactive_end/28